src/subscription_manager/gui/mysubstab.py

```diff
@@ -150,7 +150,9 @@ def _on_unsubscribe_prompt_response(self, dialog, response, selection):
             self.async_bind.unbind(serial, selection, self._unsubscribe_callback, self._handle_unbind_exception)
         else:
             # unregistered, just delete the certs directly
-            self.backend.entcertlib.delete([serial])
+            for ent in self.entitlement_dir.list():
```

Comment:
it is the logic in the CLI that does the same function. I guess it could be made common. Don't we plan to commonize the whole thing anyway?